pages/interface/components/TabCoinButtons/index.js

```diff
@@ -87,19 +87,21 @@ export default function TabCoinButtons({ content }) {
         alignItems: 'center',
         mt: contentObject.title ? '9px' : '0px',
       }}>
-      <Box>
-        <IconButton
-          variant="invisible"
-          aria-label="Creditar TabCoin"
-          icon={ChevronUpIcon}
-          size="small"
-          sx={{ color: 'fg.subtle', lineHeight: '18px' }}
-          onClick={() => {
-            transactTabCoin('credit');
-          }}
-          disabled={isPosting || isAnimatingCredit || isAnimatingDebit}
-        />
-      </Box>
+      <Tooltip aria-label="Achei relevante" direction="ne" noDelay={true}>
+        <Box>
+          <IconButton
+            variant="invisible"
+            aria-label="Creditar TabCoin"
+            icon={ChevronUpIcon}
+            size="small"
+            sx={{ color: 'fg.subtle', lineHeight: '18px' }}
+            onClick={() => {
+              transactTabCoin('credit');
+            }}
+            disabled={isPosting || isAnimatingCredit || isAnimatingDebit}
+          />
+        </Box>
+      </Tooltip>
```

Comment:
```suggestion
      <Tooltip aria-label="Achei relevante" direction="ne">
        <IconButton
          variant="invisible"
          aria-label="Creditar TabCoin"
          icon={ChevronUpIcon}
          size="small"
          sx={{ color: 'fg.subtle', lineHeight: '18px' }}
          onClick={() => transactTabCoin('credit')}
          disabled={isInAction}
        />
      </Tooltip>
```

1. Remover o `<Box>` desnecessário;
2. Atualizar para as novas versões de `onClick` e `disable`;
3. Sugiro a remoção do `noDelay={true}`;
4. Fazer o mesmo com o botão de débito.